src/cli/rustup_mode.rs

```diff
@@ -1491,7 +1520,7 @@ fn override_remove(cfg: &Cfg, m: &ArgMatches<'_>) -> Result<utils::ExitCode> {
             info!("override toolchain for '{}' removed", path);
         } else {
             info!("no override toolchain for '{}'", path);
-            if !m.is_present("path") && !m.is_present("nonexistent") {
+            if !m.contains_id("path") && !m.get_flag("nonexistent") {
```

Comment:
In some places I think it's arguably better to use `Arg::contains_id` for arguments that take value(s), instead of `.get_one::<String>().is_some()`.